### PR TITLE
Dashboard edits

### PR DESCRIPTION
**Pull request by Abbey-Wilkinson**
- Added formatting to the main app.
- Add merging functions to database.
- Changed visualisations for merged functions.

### CODE REVIEW

File: dashboard/app.py [hunk 1]
```diff
@@ -23,58 +23,63 @@
...
-def get_temperature_warning_metrics(critical_temp_plants) -> None:
+def get_temperature_warning_metrics(critical_temp) -> None:
     """
     Gets the main temperature warnings and displays them at the top.
     """
     st.warning(
-        f'WARNING! The following plants are in CRITICAL TEMPERATURE CONDITION: \n\n{critical_temp_plants}', icon="⚠️")
+        f'''WARNING! The following plants are in CRITICAL TEMPERATURE CONDITION:
+        \n\n{critical_temp}''',
+        icon="⚠️")
```
Comment (caithomas6789) [inline dashboard/app.py:33]: Good formatting.

File: dashboard/app.py [hunk 2]
```diff
@@ -23,58 +23,63 @@
                             get_soil_moisture_over_time)
 
 
-def get_temperature_warning_metrics(critical_temp_plants) -> None:
+def get_temperature_warning_metrics(critical_temp) -> None:
```
Comment (IB64) [inline dashboard/app.py:26]: Perhaps this may need some type hinting

File: dashboard/app.py [hunk 3]
```diff
@@ -99,39 +104,48 @@ def get_main_body(plants: DataFrame, merged: DataFrame) -> None:
 
     st.set_page_config(layout="wide")
 
-    st.title("Plant Sensors Dashboard")
+    st.title(":cactus: Plant Sensors Dashboard :cactus:")
```
Comment (caithomas6789) [inline dashboard/app.py:107]: I like the use of emojis.

File: dashboard/app.py [hunk 4]
```diff
@@ -99,39 +104,48 @@ def get_main_body(plants: DataFrame, merged: DataFrame) -> None:
...
+    st.sidebar.divider()
 
     selected_plants = get_selected_plants(plants)
 
     name_in_selected_plants = get_names_of_selected_plants(
         plants, selected_plants)
+    st.sidebar.divider()
 
-    critical_temp_plants = get_names_of_critical_temp_plants(plants)
+    CRITICAL_TEMP_PLANTS = get_names_of_critical_temp_plants(plants)
```
Comment (OscarB28) [inline dashboard/app.py:119]: global variables are important

**Review by OscarB28 — APPROVED** [review]

**Pull request merged**